### ISSUE TITLE
App crash from Production.

### ISSUE THREAD

**Issue by rafaelekol** [horizontalsystems/unstoppable-wallet-android#1005]
```
at android.database.sqlite.SQLiteConnection.nativePrepareStatement (Native Method)
at android.database.sqlite.SQLiteConnection.acquirePreparedStatement (SQLiteConnection.java:903)
at android.database.sqlite.SQLiteConnection.prepare (SQLiteConnection.java:514)
at android.database.sqlite.SQLiteSession.prepare (SQLiteSession.java:588)
at android.database.sqlite.SQLiteProgram.<init> (SQLiteProgram.java:58)
at android.database.sqlite.SQLiteQuery.<init> (SQLiteQuery.java:37)
at android.database.sqlite.SQLiteDirectCursorDriver.query (SQLiteDirectCursorDriver.java:46)
at android.database.sqlite.SQLiteDatabase.rawQueryWithFactory (SQLiteDatabase.java:1408)
at android.database.sqlite.SQLiteDatabase.rawQueryWithFactory (SQLiteDatabase.java:1383)
at androidx.sqlite.db.framework.FrameworkSQLiteDatabase.query (FrameworkSQLiteDatabase.java:161)
at androidx.room.RoomDatabase.query (RoomDatabase.java:305)
at io.horizontalsystems.bitcoincore.storage.PeerAddressDao_Impl.getLeastScoreFastest (PeerAddressDao_Impl.java:166)
at io.horizontalsystems.bitcoincore.storage.Storage.getLeastScoreFastestPeerAddressExcludingIps (Storage.kt:21)
at io.horizontalsystems.bitcoincore.network.peer.PeerAddressManager.getLeastScoreFastestPeer (PeerAddressManager.kt:66)
at io.horizontalsystems.bitcoincore.network.peer.PeerAddressManager.getIp (PeerAddressManager.kt:30)
at io.horizontalsystems.bitcoincore.network.peer.PeerGroup.connectPeersIfRequired (PeerGroup.kt:163)
at io.horizontalsystems.bitcoincore.network.peer.PeerGroup.onDisconnect (PeerGroup.kt:107)
at io.horizontalsystems.bitcoincore.network.peer.Peer.disconnected (Peer.kt:113)
at io.horizontalsystems.bitcoincore.network.peer.PeerConnection.run (PeerConnection.kt:77)
```

**Comment by rafaelekol** [comment]
Moved to Bitcoin Kit project

**Comment by rafaelekol** [comment]
Crash still occurs, after latest hot fix in version 0.6.3

```
at android.database.sqlite.SQLiteConnection.nativePrepareStatement (Native Method)
at android.database.sqlite.SQLiteConnection.acquirePreparedStatement (SQLiteConnection.java:1229)
at android.database.sqlite.SQLiteConnection.prepare (SQLiteConnection.java:703)
at android.database.sqlite.SQLiteSession.prepare (SQLiteSession.java:588)
at android.database.sqlite.SQLiteProgram.<init> (SQLiteProgram.java:59)
at android.database.sqlite.SQLiteQuery.<init> (SQLiteQuery.java:37)
at android.database.sqlite.SQLiteDirectCursorDriver.query (SQLiteDirectCursorDriver.java:46)
at android.database.sqlite.SQLiteDatabase.rawQueryWithFactory (SQLiteDatabase.java:1865)
at android.database.sqlite.SQLiteDatabase.rawQueryWithFactory (SQLiteDatabase.java:1840)
at androidx.sqlite.db.framework.FrameworkSQLiteDatabase.query (FrameworkSQLiteDatabase.java:161)
at androidx.room.RoomDatabase.query (RoomDatabase.java:305)
at io.horizontalsystems.bitcoincore.storage.PeerAddressDao_Impl.getLeastScoreFastest (PeerAddressDao_Impl.java:166)
at io.horizontalsystems.bitcoincore.storage.Storage.getLeastScoreFastestPeerAddressExcludingIps (Storage.kt:21)
at io.horizontalsystems.bitcoincore.network.peer.PeerAddressManager.getLeastScoreFastestPeer (PeerAddressManager.kt:67)
at io.horizontalsystems.bitcoincore.network.peer.PeerAddressManager.getIp (PeerAddressManager.kt:31)
at io.horizontalsystems.bitcoincore.network.peer.PeerGroup.connectPeersIfRequired (PeerGroup.kt:163)
at io.horizontalsystems.bitcoincore.network.peer.PeerGroup.onDisconnect (PeerGroup.kt:107)
at io.horizontalsystems.bitcoincore.network.peer.Peer.disconnected (Peer.kt:113)
at io.horizontalsystems.bitcoincore.network.peer.PeerConnection.run (PeerConnection.kt:74)
```

```
android.database.CursorWindowAllocationException:
at android.database.CursorWindow.<init> (CursorWindow.java:136)
at android.database.sqlite.SQLiteCursor.clearOrCreateWindow (SQLiteCursor.java:315)
at android.database.sqlite.SQLiteCursor.fillWindow (SQLiteCursor.java:155)
at android.database.sqlite.SQLiteCursor.getCount (SQLiteCursor.java:149)
at android.database.AbstractCursor.moveToPosition (AbstractCursor.java:220)
at android.database.AbstractCursor.moveToFirst (AbstractCursor.java:259)
at io.horizontalsystems.bitcoincore.storage.PeerAddressDao_Impl.getLeastScoreFastest (PeerAddressDao_Impl.java:172)
at io.horizontalsystems.bitcoincore.storage.Storage.getLeastScoreFastestPeerAddressExcludingIps (Storage.kt:21)
at io.horizontalsystems.bitcoincore.network.peer.PeerAddressManager.getLeastScoreFastestPeer (PeerAddressManager.kt:67)
at io.horizontalsystems.bitcoincore.network.peer.PeerAddressManager.getIp (PeerAddressManager.kt:31)
at io.horizontalsystems.bitcoincore.network.peer.PeerGroup.connectPeersIfRequired (PeerGroup.kt:163)
at io.horizontalsystems.bitcoincore.network.peer.PeerGroup.onDisconnect (PeerGroup.kt:107)
at io.horizontalsystems.bitcoincore.network.peer.Peer.disconnected (Peer.kt:113)
at io.horizontalsystems.bitcoincore.network.peer.PeerConnection.run (PeerConnection.kt:77)
```

**Comment by rafaelekol** [comment]
Fixed by https://github.com/horizontalsystems/bitcoin-kit-android/pull/392
